toolsay/toolsay.csproj

```diff
@@ -69,4 +71,29 @@
     <Delete Files="@(_PackageFilesToDelete)"/>
   </Target>
 
+  <Target Name="_AlsoPackTool">
+    <ItemGroup>
+      <_ToolProjectToPack Include="$(MSBuildProjectFullPath)" AdditionalProperties="PackAsTool=true" />
+    </ItemGroup>
+    <MSBuild Projects="@(_ToolProjectToPack)" Targets="PackTool">
+      <Output TaskParameter="TargetOutputs" ItemName="TfmSpecificPackageFile" />
+    </MSBuild>
+  </Target>
+
+  <Target Name="_AlsoSetToolProperties">
+    <PropertyGroup>
+      <PackageType>$(PackageType);DotNetTool</PackageType>
+      <PackageTags>$(PackageTags);tool</PackageTags>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="_AlsoTriggerRIDPackageBuilds">
+    <ItemGroup>
+      <_ToolProjectToTriggerRidBuilds Include="$(MSBuildProjectFullPath)" AdditionalProperties="PackAsTool=true" />
+    </ItemGroup>
+    <MSBuild Projects="@(_ToolProjectToTriggerRidBuilds)" Targets="_CreateRIDSpecificToolPackages">
+      <Output TaskParameter="TargetOutputs" ItemName="_ToolPackageOutputs" />
+    </MSBuild>
+  </Target>
```

Comment:
For multi-RID package this ensures that we fire off the RID-specific sub-package builds like would normally happen for a new-style multi-RID tool package.